scss/navbar.scss

```diff
@@ -0,0 +1,21 @@
+.nav {
+  align-items: center;
+  box-shadow: 0 0 10px $black;
+  display: flex;
+  height: 90px;
+  justify-content: space-between;
+  margin-bottom: 50px;
+  padding: 0 10px;
+  position: static;
+  top: 0;
+  z-index: 10;
+}
+
+.wlogo {
+  height: 90px;
+  width: 100px;
```

Comment:
in images set only width or height and let the other adapt unless you want to enforce a specific size but letting adapt does not deform the image.